src/classes/REL_Relationships_TEST.cls

```diff
@@ -385,24 +438,22 @@ public class REL_Relationships_TEST {
         setupRelationshipTestData(new Hierarchy_Settings__c(Reciprocal_Method__c = 'Value Inversion'));
 
         Relationship__c[] crel = new Relationship__c[]{
-            new Relationship__c(contact__c = c1.id, Relatedcontact__c = c2.id, Type__c='Buddies-Friends')
+            new Relationship__c(Contact__c = c1.Id, RelatedContact__c = c2.Id, Type__c='Buddies-Friends')
         };
-        //reset the process controller to mimic a second user action
-        //Relationships_ProcessControl.hasRun = false;
 
         Test.startTest();
         insert crel;
         Test.stopTest();
 
         // check that rel gets created
-        Relationship__c crel_original = [select id, Description__c,Type__c,reciprocalrelationship__c from Relationship__c where id=:crel[0].Id];
+        Relationship__c crel_original = [SELECT Id, Description__c,Type__c,ReciprocalRelationship__c FROM Relationship__c WHERE Id=:crel[0].Id];
```

Comment:
spacing here. Would you rather during reviews, when I find little items like this that I just fix 'em right here via github rather than bother you with the mundane?

Reply:
@lifewithryan I don't mind fixing them. I would have been more thorough on this initially but was just trying to do the quick and easy stuff :)

Reply:
No worries. Just figured since you were in there.